openupgrade_scripts/scripts/point_of_sale/14.0.1.0.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,97 @@
+point_of_sale / pos.payment              / payment_status (char)         : NEW
+point_of_sale / pos.payment              / ticket (char)                 : NEW
+point_of_sale / pos.payment.method       / active (boolean)              : NEW hasdefault
+point_of_sale / pos.session              / cash_real_difference (float)  : NEW
+point_of_sale / pos.session              / cash_real_expected (float)    : NEW
+point_of_sale / pos.session              / cash_real_transaction (float) : NEW
+point_of_sale / pos.session              / picking_ids (one2many)        : NEW relation: stock.picking
+point_of_sale / pos.session              / state (selection)             : selection_keys is now '['closed', 'closing_control', 'opened', 'opening_control']' ('['closed', 'closing_control', 'new_session', 'opened', 'opening_control']')
+point_of_sale / pos.session              / update_stock_at_closing (boolean): NEW
+# NOTHING TO DO
+
+point_of_sale / res.company              / point_of_sale_update_stock_quantities (selection): NEW selection_keys: ['closing', 'real'], hasdefault
+# Done: post-migration set value to 'real' corresponding v13 work
+
+point_of_sale / res.partner              / barcode (char)                : module is now 'base' ('point_of_sale')
+point_of_sale / res.partner              / barcode (char)                : not stored anymore
+# NOTHING TO DO
+
+point_of_sale / stock.picking            / pos_order_id (many2one)       : NEW relation: pos.order
+# DONE: post-migration convert data many2one to one2many
+
+point_of_sale / stock.picking            / pos_session_id (many2one)     : NEW relation: pos.session
+# NOTHING TO DO: new feature
+
+pos_cash_rounding / account.cash.rounding    / loss_account_id (many2one)    : module is now 'account' ('pos_cash_rounding')
+pos_cash_rounding / account.cash.rounding    / loss_account_id (many2one)    : not stored anymore
+# NOTHING TO DO: handle by ORM
+
+---XML records in module 'point_of_sale'---
+NEW ir.model.access: point_of_sale.access_account_cash_rounding_pos_user
+NEW ir.model.access: point_of_sale.access_closing_balance_confirm_wizard
+NEW ir.model.access: point_of_sale.access_money_in_out_wizard
+NEW ir.model.access: point_of_sale.access_pos_details_wizard
+NEW ir.model.access: point_of_sale.access_pos_make_payment
+NEW ir.model.access: point_of_sale.access_product_attribute_custom_value_pos_manager
+NEW ir.model.access: point_of_sale.access_product_attribute_pos_manager
+NEW ir.model.access: point_of_sale.access_product_attribute_value_pos_manager
+NEW ir.model.access: point_of_sale.access_product_product_attribute_pos_manager
+NEW ir.model.access: point_of_sale.access_product_template_attribute_exclusion_pos_manager
+NEW ir.model.access: point_of_sale.access_product_template_attribute_line_pos_manager
+DEL ir.model.access: pos_cash_rounding.access_account_cash_rounding
+DEL ir.model.constraint: point_of_sale.constraint_res_partner_unique_barcode
```

Comment:
you have to delete it. see https://github.com/OCA/OpenUpgrade/commit/8756d0413ee2794bdb737300a1de6bf9b46453c5 as example